README.md

```diff
@@ -124,6 +124,16 @@ ultimately appear on NPM.
 2. Run `yarn prepack` to build all packages
 3. Run `yarn lint` to run linters
 
+### Typescript Project References
```

Comment:
Let's get rid of this sub-heading and move the content of the section up into the "Building" section above since this is a natural extension of building.... If we do want to separate it into two parts, then maybe a "building during development"

However we do it, I think that these sections should be adjacent.